# HG changeset patch
# User Andrew Cooper <andrew.cooper3@citrix.com>
# Date 1383833745 -3600
# Node ID 01266f0fa51330f06932231c53041653f57f7eff
# Parent  ec825756ace20159408baabfc0075d7fa8945fca
x86/hpet: Fix ambiguity in broadcast info message

"$N will be used for broadcast" is ambiguous between "$N timers" or "timer
$N", particuarly when N is 0.

Signed-off-by: Andrew Cooper <andrew.cooper3@citrix.com>
Reviewed-by: Tim Deegan <tim@xen.org>

diff -r ec825756ace2 -r 01266f0fa513 xen/arch/x86/hpet.c
--- a/xen/arch/x86/hpet.c
+++ b/xen/arch/x86/hpet.c
@@ -430,8 +430,8 @@ static void __init hpet_fsb_cap_lookup(v
             num_hpets_used++;
     }
 
-    printk(XENLOG_INFO "HPET: %u timers (%u will be used for broadcast)\n",
-           num_chs, num_hpets_used);
+    printk(XENLOG_INFO "HPET: %u timers usable for broadcast (%u total)\n",
+           num_hpets_used, num_chs);
 }
 
 static struct hpet_event_channel *hpet_get_channel(unsigned int cpu)
